src/model.js

```diff
@@ -119,23 +117,24 @@ export default class Model {
         /*
          Для массивов объектов из нового и старого файлов моделей
          вызываем метод _generateUrlPageMap, который строит из данных массивов
-         объекты в которых ключами являются url страниц, а значениями сами объекты
+         объекты, в которых ключами являются url страниц, а значениями сами объекты
          */
-        const newModel = this.constructor._generateUrlPageMap(currentModel);
-        const oldModel = this.constructor._generateUrlPageMap(previousModel);
+        const generateUrlPageMap = this.constructor._generateUrlPageMap;
+        const newModel = generateUrlPageMap(currentModel);
+        const oldModel = generateUrlPageMap(previousModel);
 
         const newPages = _.keys(newModel); // получить все url из новой модели
         let oldPages = _.keys(oldModel); // получить все url из старой модели
 
         /*
          Добавленные страницы можно получить вычислив разницу между массивом url из новой и старой моделей
-         Для удаленных страницы наоборот надо вычислить разницу между массивом url из старой и новой моделей
+         Для удаленных страниц наоборот надо вычислить разницу между массивом url из старой и новой моделей
          */
         const addedPages = _.difference(newPages, oldPages);
         const removedPages = _.difference(oldPages, newPages);
 
         removedPages.forEach(url => {
-            this.pushChangeRemove({type: 'page', url});
+            this.pushChangeToRemovedGroup({type: 'page', url});
```

Comment:
'object short notation' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).